docs/reporting/key-concepts.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Key Concepts
+description: Geocortex Reporting - Learn about key concepts of Geocortex Reporting
+hide_table_of_contents: true
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import DesignerCallout from "./snippets/designer-callout.mdx";
+
+<DesignerCallout />
+
+## Reports
+
+A report is a formatted presentation of a particular type of information. Geocortex Reporting specializes in presenting data and images related to web maps.
+
+Reports are designed by report authors (administrators) using a web-based tool called Geocortex Report Designer. They are saved to the Geocortex Reporting service, with an item in the content area of the author's ArcGIS account. To allow users to run a report, the author shares the ArcGIS item with them. End users use the reports designed by the author to create on-demand, print-ready reports about maps and their features.
+
+End users run reports from web applications that the author has configured to host the reports (host applications). Alternatively, a report can be run using the Geocortex Reporting REST API. The output report is a PDF.
+
+## Geocortex Report Designer
+
+Geocortex Report Designer provides an extensive suite of tools for laying out reports, specifying the data that you want to include, and presenting the data using charts, tables, and a wide variety of other configurable elements. Organizations can sign into the cloud-hosted report designer, or they can host it themselves on their own web server. They can then author as many report templates as their application requirements demand and save them into ArcGIS Online or Portal for ArcGIS as items. Sharing reports with end users allows them to create on-demand, print-ready PDF reports about maps and features, using data from the user's current session.
+
+Report Designer has a built-in Preview environment for running reports. Using the Preview is the quickest way to test a report while you are designing it.
+
+## Geocortex Reporting Service
+
+The [Geocortex Reporting REST API](https://apps.geocortex.com/reporting/service/api-viewer/index.html?url=../swagger.json#/) can be used to asynchronously run a report, watch for report completion and download the completed report as a PDF.
+
+The basic steps for running a report are:
+
+1. Obtain a run token from the [/auth/token/run](https://apps.geocortex.com/reporting/service/api-viewer/index.html?url=../swagger.json#/auth/post_auth_token_run) endpoint.
```

Comment:
Should we add a sub-bullet to say this is only required if the report is secured or if the reports access secured content?

Reply:
Done